### ISSUE TITLE
unpatch: better document /commands

### ISSUE THREAD

**Issue by reduzent** [reduzent/netpd#12]
What does `/dev on` do again? It does protect unpatch from other clients opening/closing instruments. Why is it called `dev` then?

**Comment by reduzent** [comment]
Current unpatch version in dev doesn't have a symbol box anymore, but a scroll  list. That also means that written commands are not available right now.

**Comment by reduzent** [comment]
I find the scroll list tedious.. I already miss the symbol box.

**Comment by reduzent** [comment]
All the irc style commands are handled in chat, except `/new instrument`. The a list of commands can be shown with `/help`. Each command also has a help.
